doc/content/verification_and_validation/ver-1jb.md

```diff
@@ -1,9 +1,142 @@
+for the decay of tritium and associated growth of $^3$He in a distributed trap.
+TMAP8 matches the analytical solution, with a root mean square percentage error
+(RMSPE) of 1.29% and 0.35% for the $I_{tot}$ and $I_{He}$ concentration curves, respectively,
+and can also provide the trapped and mobile tritium concentrations.
+
+!media comparison_ver-1jb.py
+       image_name=ver-1jb_comparison_analytical_time_evolution.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1jb_results_comparison_analytical_time_evolution_1
+       caption=Comparison of TMAP8 predictions against the analytical solution for the decay of tritium and associated growth of $^3$He in a distributed trap with a small concentration of mobile tritium compared to trapped tritium.
+
+[ver-1jb_results_profile_1] shows the depth profile of the initial trapped atoms of tritium, the concentration of trapped atoms of
+tritium after 45 years, and the distribution of $^3$He at the end of that time across the distributed trap as predicted by TMAP8.
+
+Note that because of $^3$He is given a null diffusivity in this verification problem, the shape of the $^3$He does not broaden.
+This could be easily implemented with existing TMAP8 diffusive capabilities, as was shown in many other TMAP8 cases,
+including [ver-1d](ver-1d.md).
+
+!media comparison_ver-1jb.py
+       image_name=ver-1jb_profile.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1jb_results_profile_1
+       caption=Concentration profiles of initially trapped tritium that decayed to $^3$He over 45 years with a small concentration of mobile tritium compared to trapped tritium.
+
+#### With equivalent mobile and trapped tritium initial concentrations
+
+[ver-1jb_results_comparison_analytical_time_evolution_2] and [ver-1jb_results_profile_2] show the results of the simulations when the initial concentrations of mobile and trapped tritium are equivalent.
+[ver-1jb_results_comparison_analytical_time_evolution_2] shows TMAP8's time predictions of the inventories and how they compare to the analytical solution. The RMSPE values as low as in the previous case.
+
+!media comparison_ver-1jb.py
+       image_name=ver-1jb_equivalent_concentrations_comparison_analytical_time_evolution.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1jb_results_comparison_analytical_time_evolution_2
+       caption=Comparison of TMAP8 predictions against the analytical solution for the decay of tritium and associated growth of $^3$He in a distributed trap with equivalent initial concentrations of mobile and trapped tritium.
+
+[ver-1jb_results_profile_2] shows the depth profile of the initial trapped atoms of tritium, the concentration of mobile and trapped atoms of
+tritium after 45 years, and the distribution of $^3$He at the end of that time across the distributed trap as predicted by TMAP8.
+
+!media comparison_ver-1jb.py
```

Comment:
There's 2 lines with the same legend in this figure (C_T(t=0)). I think you maybe want to specify that one line is trapped and one is mobile?

Reply:
Again, great catch! Done.